development.md

```diff
@@ -0,0 +1,24 @@
+### Get  Started with Development 
+#
+
+## prerequisites
+
+- installed go if not installed. Follow the below link for more details
+
+[go installation] (https://go.dev/dl/)
+
+- git should be installed for cloning 
+
+- usage of ide. Any ide can be used but visual studio is preferable 
```

Comment:
We shouldn't list an IDE as a prerequisite.

Reply:
remove ide part. But i think usage of ide also can be suggested somewhere so that the contributer can get a head start